src/UI/Input.jsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable */
 function Input({ type, placeholder }) {
   return (
```

Comment:
[OPTIONAL] Kindly add MIT license to your project.

Reply:
This change has been done well.